src/Symfony/Component/BrowserKit/Client.php

```diff
@@ -68,6 +69,16 @@ public function followRedirects($followRedirect = true)
         $this->followRedirects = (bool) $followRedirect;
     }
 
+    /**
+     * Sets whether to automatically follow meta refresh redirects or not.
+     *
+     * @param bool $followMetaRefresh Whether to follow meta refresh redirects
```

Comment:
the description of the argument is not really needed

Reply:
I've removed the description.